migrations/20201024121554_alter_record_clear_time.mysql.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE records DROP COLUMN min_clear_time;
+ALTER TABLE records ADD COLUMN clear_time float COMMENT '最短クリア時間';
```

Comment:
```suggestion
ALTER TABLE records ADD COLUMN clear_time FLOAT COMMENT '最短クリア時間';
```
同値なはずだけど，今までの書き方が全大文字表現なので，合わせてもらえますか？（ `BOOLEAN` ， `INT` など）

Reply:
後，時間の単位を書いておいてもらっていいですか？

```suggestion
ALTER TABLE records ADD COLUMN clear_time float COMMENT '最短クリア時間 (秒)';
```
など